src/test/java/org/broadinstitute/consent/http/resources/DatasetResourceTest.java

```diff
@@ -656,7 +653,7 @@ public void testUpdateNeedsReviewDataSetsError() {
 
     @Test
     public void testDownloadDatasetApprovedUsersSuccess() {
-        List header = List.of("attachment; filename =DatasetApprovedUsers.tsv");
+        List<String> header = List.of("attachment; filename =DatasetApprovedUsers.tsv");
```

Comment:
This is actually replicating a bug we have defined somewhere in the backlog. The space in `filename =DatasetApprovedUsers.tsv` breaks the actual download.

Reply:
That's odd. There's no space in the header definition, but the test shows that it's there in the response. It fails when there is no space. Should I do something about this?

Reply:
Yes, it needs to be fixed. 
See: 
* https://stackoverflow.com/questions/93551/how-to-encode-the-filename-parameter-of-content-disposition-header-in-http
* https://broadworkbench.atlassian.net/browse/DUOS-1319

It looks like 1319 is resolved, but the test here is clearly not working as it should. There should be no space between`filename` and `=`.